doc/whats_new.rst

```diff
@@ -19,20 +19,37 @@ Here we list a changelog of MNE-connectivity.
 Version 0.8 (in dev)
 --------------------
 
+Functionality now requires ``Python >= 3.10``.
+
 Enhancements
 ~~~~~~~~~~~~
 
-
+- Add a new :class:`~mne_connectivity.decoding.CoherencyDecomposition` class for decomposing connectivity sources using multivariate coherency-based methods, by `Thomas Binns`_ (:pr:`193`).
+- Add new plotting methods :meth:`~mne_connectivity.decoding.CoherencyDecomposition.plot_filters` and :meth:`~mne_connectivity.decoding.CoherencyDecomposition.plot_patterns` to the :class:`~mne_connectivity.decoding.CoherencyDecomposition` class for visualising the decomposed connectivity sources, by `Thomas Binns`_ (:pr:`208`).
```

Comment:
It's fine as-is, but FYI you could hack the cross-refs to be a bit more contextually informative, like this:

```suggestion
- Add new plotting methods :meth:`CoherencyDecomposition.plot_filters() <mne_connectivity.decoding.CoherencyDecomposition.plot_filters>` and :meth:`CoherencyDecomposition.plot_patterns() <mne_connectivity.decoding.CoherencyDecomposition.plot_patterns>` for visualising the decomposed connectivity sources, by `Thomas Binns`_ (:pr:`208`).
```